app/models/user.rb

```diff
@@ -11,7 +11,7 @@ class User
   attr_accessor :email, :access_token, :challenge_name, :cognito_session_id,
                 :mfa, :challenge_parameters, :roles, :full_name, :family_name,
                 :given_name, :phone_number, :user_id, :login_id, :password, :new_password,
-                :totp_code, :permissions, :session_start_time, :team, :cognito_groups
+                :totp_code, :permissions, :session_start_time, :team, :cognito_groups, :secret_code
```

Comment:
`:secret_code` can be added to the `filter_parameter_logging.rb`

Reply:
Added username and secret code to the list of filter_parameters_logging